package.json

```diff
@@ -27,30 +27,40 @@
     "helm:build": "mkdir -p tmp && helm package -d tmp deployment/forklift-console-plugin/ && npm run helm:index"
   },
   "devDependencies": {
+    "@openapitools/openapi-generator-cli": "^2.5.2",
+    "@rollup/plugin-commonjs": "^24.0.1",
```

Comment:
is rollup needed for API generation?

Reply:
no it's not needed for the api generation, it's part of packaging the types package at the end of the proccess, I'll add it as a 4'th commit :-)